prisma/migrations/20240602203851_init/migration.sql

```diff
@@ -0,0 +1,93 @@
+-- CreateEnum
+CREATE TYPE "AuthType" AS ENUM ('PASSWORD', 'GITHUB');
+
+-- CreateEnum
+CREATE TYPE "EntityType" AS ENUM ('PERSON', 'ORGANIZATION', 'CLAIM', 'IMPACT', 'EVENT', 'DOCUMENT', 'PRODUCT', 'PLACE', 'UNKNOWN', 'OTHER');
+
+-- CreateEnum
+CREATE TYPE "IssuerIdType" AS ENUM ('DID', 'ETH', 'PUBKEY', 'URL');
+
+-- CreateEnum
+CREATE TYPE "HowKnown" AS ENUM ('FIRST_HAND', 'SECOND_HAND', 'WEB_DOCUMENT', 'VERIFIED_LOGIN', 'BLOCKCHAIN', 'SIGNED_DOCUMENT', 'PHYSICAL_DOCUMENT', 'INTEGRATION', 'RESEARCH', 'OPINION', 'OTHER');
+
+-- CreateTable
+CREATE TABLE "User" (
+    "id" SERIAL NOT NULL,
+    "email" TEXT,
+    "passwordHash" TEXT,
```

Comment:
we already have a user table i think?  note that sometimes, claims are created by the server, or we might read them off a feed later signed by some DID who is NOT a user on our system